celery/beat/__init__.py

```diff
@@ -26,14 +26,14 @@
 from kombu.utils import cached_property, reprcall
```

Comment:
Converting a module to a package (or the other way around) can cause problems if the user installs the new version on top of the old one (since the old module will not be deleted, and .pyc files may also exist).

So something should be added to https://github.com/celery/celery/blob/master/setup.py#L17-L46, but upgrading instead of downgrade_packages.